apps/frontend/src/pages/overview/libs/components/add-course-modal/add-course-modal.tsx

```diff
@@ -103,7 +112,7 @@ const AddCourseModal: React.FC<Properties> = ({ onClose }: Properties) => {
 						/>
 						<div className={styles["toolbar"]}>
 							<p className={styles["results-count"]}>
-								{courses.length} results
+								{courses.length + recommendedCourses.length} results
```

Comment:
```suggestion
								{courses.length} results
```
Recommended courses are the same as other courses, but sorted